distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -47,8 +47,6 @@ log4j2.logger.securityLogger.appenderRef.osgi-platformLogging.ref = osgi-platfor
 log4j2.logger.org_apache_karaf_jaas_modules_audit.name = org.apache.karaf.jaas.modules.audit
 log4j2.logger.org_apache_karaf_jaas_modules_audit.level = INFO
 log4j2.logger.org_apache_karaf_jaas_modules_audit.additivity = false
-# TODO DDF-3073 (see note below)
-#log4j2.logger.org_apache_karaf_jaas_modules_audit.appenderRef.securityFailover.ref = securityFailover
 log4j2.logger.org_apache_karaf_jaas_modules_audit.appenderRef.securityMain.ref = securityMain
```

Comment:
This line should be removed because `securityMain` is the primary appender of the `securityFailover` Failover.